spec.html

```diff
@@ -45037,7 +45037,7 @@ <h1>
         <h1>
           AsyncGeneratorYield (
             _value_: unknown,
-          ): either a normal completion containing an ECMAScript language value or an abrupt completion
+          ): either a normal completion containing an ECMAScript language value or a throw completion
```

Comment:
This is also wrong, presumably because of AsyncGeneratorUnwrapYieldResumption being wrong.